src/main/scala/com/example/PersistentSagaActor.scala

```diff
@@ -154,7 +154,7 @@ class PersistentSagaActor(persistentEntityRegion: ActorRef)
                 pendingTransitionCheck()
               }
             case _: TransactionException =>
-              if (!state.exceptions.contains(envelope.entityId)) {
```

Comment:
Thanks Renato, I just saw this PR. I have already heavily refactored and applied this change. I need more test coverage...